dashboard/app/helpers/levels_helper.rb

```diff
@@ -265,7 +265,7 @@ def render_app_dependencies
     use_droplet = app_options[:droplet]
     use_netsim = @level.game == Game.netsim
     use_applab = @level.game == Game.applab
-    use_gamelab = @level.game == Game.gamelab
+    use_gamelab = @level.game.app == Game::GAMELAB
```

Comment:
To capture what we talked about in person — elsewhere `game` is almost always synonymous with `app`.  To the point that we'd been thinking about getting rid of `game` entirely in favor of `app`.

This would be the first place we use `app` to differentiate two different level types that use the same `game`.  Not necessarily a bad thing, but worth thinking through if this is the best way to do it.

Reply:
In addition to `game` and `app`, we also have a level's `type`, and we probably don't need all 3. I'm pretty sure just using the gamelab_jr level type with the normal gamelab game would work too, so we could probably merge `app` and `game`.

Out of curiousity, I pulled all 81 level type/game/app combinations we have in the DB:
```
mysql> SELECT levels.type,games.name AS game,games.app FROM levels,games WHERE games.id=levels.game_id GROUP BY games.name,games.app,levels.type;
+-----------------------+-----------------------+-------------------------+
| type                  | game                  | app                     |
+-----------------------+-----------------------+-------------------------+
| Applab                | Applab                | applab                  |
| Blockly               | Applab                | applab                  |
| Blockly               | Artist                | turtle                  |
| Blockly               | Artist2               | turtle                  |
| Blockly               | Artist3               | turtle                  |
| NULL                  | Artist4               | turtle                  |
| Blockly               | Artist4               | turtle                  |
| Blockly               | Artist5               | turtle                  |
| Blockly               | ArtistEC              | turtle                  |
| Blockly               | Bounce                | bounce                  |
| Bounce                | Bounce                | bounce                  |
| Blockly               | Calc                  | calc                    |
| Calc                  | Calc                  | calc                    |
| ContractMatch         | ContractMatch         | NULL                    |
| Craft                 | Craft                 | craft                   |
| CurriculumReference   | CurriculumReference   | curriculum_reference    |
| Map                   | CurriculumReference   | curriculum_reference    |
| NULL                  | Custom                | turtle                  |
| Artist                | Custom                | turtle                  |
| Flappy                | CustomFlappy          | flappy                  |
| NULL                  | CustomMaze            | maze                    |
| Blockly               | CustomMaze            | maze                    |
| Karel                 | CustomMaze            | maze                    |
| Maze                  | CustomMaze            | maze                    |
| StarWarsGrid          | CustomStudio          | studio                  |
| Studio                | CustomStudio          | studio                  |
| Blockly               | Eval                  | eval                    |
| Eval                  | Eval                  | eval                    |
| EvaluationMulti       | EvaluationMulti       | evaluation_multi        |
| External              | External              | external                |
| ExternalLink          | ExternalLink          | external_link           |
| NULL                  | Farmer                | maze                    |
| Blockly               | Farmer                | maze                    |
| Blockly               | Farmer2               | maze                    |
| Blockly               | Farmer3               | maze                    |
| Blockly               | Flappy                | flappy                  |
| FreeResponse          | FreeResponse          | free_response           |
| ScriptCompletion      | FreeResponse          | free_response           |
| FrequencyAnalysis     | FrequencyAnalysis     | frequency_analysis      |
| Gamelab               | Gamelab               | gamelab                 |
| Blockly               | Jigsaw                | jigsaw                  |
| LevelGroup            | LevelGroup            | level_group             |
| Match                 | Match                 | match                   |
| Blockly               | Maze                  | maze                    |
| NULL                  | MazeEC                | maze                    |
| Blockly               | MazeEC                | maze                    |
| Blockly               | MazeStep              | maze                    |
| Blockly               | Multi                 | multi                   |
| Multi                 | Multi                 | multi                   |
| Blockly               | NetSim                | netsim                  |
| NetSim                | NetSim                | netsim                  |
| Odometer              | Odometer              | odometer                |
| Pixelation            | Pixelation            | pixelation              |
| PublicKeyCryptography | PublicKeyCryptography | public_key_cryptography |
| Scratch               | Scratch               | scratch                 |
| StandaloneVideo       | StandaloneVideo       | standalone_video        |
| Blockly               | Studio                | studio                  |
| Studio                | Studio                | studio                  |
| Blockly               | StudioEC              | studio                  |
| Studio                | StudioEC              | studio                  |
| StudioEC              | StudioEC              | studio                  |
| TextCompression       | TextCompression       | text_compression        |
| TextMatch             | TextMatch             | NULL                    |
| Blockly               | Unplug1               | unplug                  |
| NULL                  | Unplug10              | unplug                  |
| Blockly               | Unplug10              | unplug                  |
| Blockly               | Unplug11              | unplug                  |
| Blockly               | Unplug2               | unplug                  |
| Blockly               | Unplug3               | unplug                  |
| NULL                  | Unplug4               | unplug                  |
| Blockly               | Unplug4               | unplug                  |
| Blockly               | Unplug5               | unplug                  |
| Blockly               | Unplug6               | unplug                  |
| NULL                  | Unplug7               | unplug                  |
| Blockly               | Unplug7               | unplug                  |
| Blockly               | Unplug8               | unplug                  |
| Blockly               | Unplug9               | unplug                  |
| Unplugged             | Unplugged             | unplug                  |
| Vigenere              | Vigenere              | vigenere                |
| Blockly               | Webapp                | webapp                  |
| Weblab                | Weblab                | weblab                  |
+-----------------------+-----------------------+-------------------------+
```

Reply:
Got rid of the new GamelabJr Game.